src/App.test.js

```diff
@@ -185,10 +184,11 @@ describe('App component', () => {
     });
 
     describe('not authenticated user', () => {
-      it('should be redirected to Legacy Doppler Login after open /reports (when using RedirectToLegacyLoginFactory)', () => {
+      it('should be redirected to Legacy Doppler Login after open /reports when useLegacy.login is active', () => {
```

Comment:
❤️  I do not want to be presumptuous, but I am very proud because this test has not required significative changes.